conan/api/subapi/cache.py

```diff
@@ -100,26 +103,28 @@ def sign(self, package_list):
                 try:
                     pkg_signer.sign_pkg(rref, recipe_bundle.get("files", {}), rref_folder)
                 except Exception as e:
-                    recipe_bundle["error"] = e
+                    recipe_bundle["pkgsign_error"] = str(e)
```

Comment:
Why not just `"error"` ? The string is what is going to provide the information about what type of error it is.